config/jobs/kubernetes/sig-testing/coverage.yaml

```diff
@@ -144,8 +144,8 @@ periodics:
       args:
       - -c
       - |
-        local result=0
-        GO111MODULE=off go install k8s.io/kubernetes/vendor/gotest.tools/gotestsum || result=$?
+        result=0
+        ( cd hack/tools && GO111MODULE=on go install gotest.tools/gotestsum ) || result=$?
```

Comment:
does this need to work for multiple versions? a temp dir and a go get would be less entangled with the guts of k/k